{"uid":"892ecabe1f29cb96","name":"test_change_country_invalid_data[invalid_payload1-authorize_admin]","fullName":"tests.orders.test_group_order_change_country#test_change_country_invalid_data","historyId":"cac25854444d493bf599a1cca2b0b56d","time":{"start":1744848864733,"stop":1744848865156,"duration":423},"description":"Негативные тесты: передача некорректных данных","descriptionHtml":"<p>Негативные тесты: передача некорректных данных</p>\n","status":"skipped","statusMessage":"XFAIL https://ru.yougile.com/team/01c2a92f1328/#IT-507\n\nAssertionError: Ожидался статус 400, но получен 405","statusTrace":"api_client = <api_tests.utils.api_client.APIClient object at 0x7f7a5098ad90>\nauthorize_func = <function authorize_admin at 0x7f7a50e1a5c0>\ninvalid_payload = {'country_id': 1, 'int_id': None}\n\n    @pytest.mark.xfail(reason = \"https://ru.yougile.com/team/01c2a92f1328/#IT-507\")\n    @pytest.mark.parametrize(\"authorize_func\", [authorize_client, authorize_partner, authorize_admin])\n    @pytest.mark.parametrize(\"invalid_payload\", [\n        {},  # Пустой запрос\n        {\"int_id\": None, \"country_id\": 1},  # Нет ID заказа\n        {\"int_id\": 1, \"country_id\": None},  # Нет ID страны\n    ])\n    def test_change_country_invalid_data(api_client, authorize_func, invalid_payload):\n        \"\"\"Негативные тесты: передача некорректных данных\"\"\"\n        authorize_func(api_client)\n        response = api_client.post(\"/orders/group_order/change_country\", json=invalid_payload)\n>       assert_response_status_and_error(response, 400)\n\napi_tests/tests/orders/test_group_order_change_country.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = {'error': '', 'status_code': 405}, expected_status = 400\n\n    def assert_response_status_and_error(response, expected_status):\n        actual_status = response[\"status_code\"]\n>       assert actual_status == expected_status, f\"Ожидался статус {expected_status}, но получен {actual_status}\"\nE       AssertionError: Ожидался статус 400, но получен 405\n\napi_tests/utils/asserts.py:39: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"api_client","time":{"start":1744848860014,"stop":1744848860014,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"api_client::0","time":{"start":1744848868896,"stop":1744848868896,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='https://ru.yougile.com/team/01c2a92f1328/#IT-507')"},{"name":"parentSuite","value":"tests.orders"},{"name":"suite","value":"test_group_order_change_country"},{"name":"host","value":"fv-az1919-435"},{"name":"thread","value":"1933-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.orders.test_group_order_change_country"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"authorize_func","value":"<function authorize_admin at 0x7f7a50e1a5c0>"},{"name":"invalid_payload","value":"{'int_id': None, 'country_id': 1}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.xfail(reason='https://ru.yougile.com/team/01c2a92f1328/#IT-507')"]},"source":"892ecabe1f29cb96.json","parameterValues":["<function authorize_admin at 0x7f7a50e1a5c0>","{'int_id': None, 'country_id': 1}"]}